product_docs/docs/biganimal/release/overview/deployment_options/index.mdx

```diff
@@ -17,3 +17,8 @@ BigAnimal's cloud account offers a seamless deployment option if you don't want
 !!! Note Note
 Currently, BigAnimal's cloud account is offered only with AWS as your cloud provider.
 !!!
+
+### Postgres distributions
+
+In order for us to effectively manage the cloud resources and ensure the Postgres distributions are protected against security threats, BigAnimal's cloud account provides specially built Postgres distributions. The Postgres distributions have a special administrative role, pg_ba_admin, and a special user that is a member of that role, ba_admin_user, which has sufficient privileges to allow your admin to effectively maintain your database. See [BigAnimal's cloud account ba_admin_user](postgres_admin_user) for details.
```

Comment:
> ensure the Postgres distributions are protected against security threats

It's users, not the distributions, that are protected.

Reply:
> BigAnimal's cloud account provides specially built Postgres distributions

Overloading "distribution" is odd. Also, these builds are going to BYOA, too.

Reply:
If these builds, and the new role and user, are not specific to a deployment option, I have put this content in the wrong location in the docs. I think the we should be talking about the new role and user in [Managing Postgres access](https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/01_postgres_access/).

Reply:
I've moved the content and hopefully fixed the confusion around deployment type